src/loaddb/load_db.c

```diff
@@ -1183,6 +1183,26 @@ ldr_server_load (load_args * args, int *status, bool * interrupted)
 		     last_stat.rows_committed, last_stat.rows_failed);
     }
 
+  // Update class statistics
+  error_code = loaddb_update_stats ();
+  if (error_code != NO_ERROR)
+    {
+      print_er_msg ();
+      *status = 3;
+    }
+
+  // Fetch the latest stats.
+  error_code = loaddb_fetch_stats (stats);
```

Comment:
it would be better if loaddb_update_stats would also fetch the stats automatically